File: algorithm/Greedy/B_changes.java
public class B_changes {
    public static void main(String[] args) {
        int money = 1260;
        int countCoin = 0;
        int[] coinTypes = {500, 100, 50, 10};

        for (int i = 0; i < 4; i++) {
            countCoin += money / coinTypes[i];
            money %= coinTypes[i];
        }

        System.out.println(countCoin);
    }
}

File: algorithm/Greedy/B_changes.py
import sys


if __name__ == "__main__":
    money = int(sys.stdin.readline().rstrip())
    
    list_coin = [500, 100, 50, 10]
    count_coin = 0

    for coin in list_coin:
        count_coin += money // coin
        money %= coin
    
    print(count_coin)

